pkg/scheduler/apis/config/v1beta2/defaults.go

```diff
@@ -97,6 +97,13 @@ func setDefaults_KubeSchedulerProfile(prof *v1beta2.KubeSchedulerProfile) {
 			Args: runtime.RawExtension{Object: args},
 		})
 	}
+	for _, e := range prof.Plugins.Score.Enabled {
```

Comment:
@damemi are the changes made now makes sense. 
- I did not added the 
`if int64(f.scorePluginWeight[e.Name])*framework.MaxNodeScore > framework.MaxTotalScore-totalPriority {
			return nil, fmt.Errorf("total score of Score plugins could overflow")
}`
condition because it was not returning values.
- I also removed `frameworkImpl` struct, as changes should be made against the type.